conda/environment.yml

```diff
@@ -9,10 +9,14 @@ channels:
 dependencies:
   - cookiecutter
   - ipyleaflet
+  - jinja2>=2.11
   - jupyter-book=0.12
   - jupyterhub-singleuser
   - jupyterlab=3
   - jupyter-resource-usage
   - mamba
+  - markdown
```

Comment:
The error message from the GH action is misleading and it was referring to the markdown extension. So we don't need this package in the environment.

Reply:
Yeah, after I saw your last comment I figured that, so it'll be removed in my next push. Thanks for the help.